model_bakery/utils.py

```diff
@@ -3,8 +3,6 @@
 import itertools
 import warnings
 
-from .timezone import tz_aware
```

Comment:
Hm, I'm not sure if we can remove the `tz_aware` module. By doing that, model bakery will stop to respect [Django's settings USE_TZ](https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-USE_TZ) and always use regular `datetime` objects.